### PR TITLE
ddbg no longer exists, website dead

### PR DESCRIPTION
**Pull request by rainers**
I guess it is left over from D1

### CODE REVIEW

**Review by thewilsonator — APPROVED** [review]

**Review by WalterBright — APPROVED** [review]

**Review by Geod24 — APPROVED** [review]

**Comment by rainers** [comment]
Unfortunately, master seems to be broken on DAutoTest with
```
ERROR: dmd/frontend.d(147): Deprecation: function `std.algorithm.iteration.each!(addGlobalIdent).each!(const(string[])).each` is deprecated
```
I cannot reproduce this locally and don't see any related recent changes. Anyone know what's going on? @CyberShadow ?

**Comment by CyberShadow** [comment]
It looks like a false positive cache hit. The dependency graph between D components has kept getting muddier with time, e.g. I see DMD now depends on Phobos. Also it seems to be ignoring the `HOST_DMD` makefile parameter. CC @wilzbach

**Comment by wilzbach** [comment]
> e.g. I see DMD now depends on Phobos

Only the build script.

> CC @wilzbach

Sorry, I don't know this either. Maybe force-push to rerun the CI?

**Comment by CyberShadow** [comment]
> Only the build script.

It doesn't look like it (thus the build error):

https://github.com/dlang/dmd/blob/master/src/dmd/frontend.d#L113

> Maybe force-push to rerun the CI?

The failure is on master, and we can't force-push that. We'd need to manually merge something else.

File: src/dmd/cli.d [hunk 1]
```diff
@@ -336,10 +336,8 @@ dmd -cov -unittest myprog.d
         Option("g",
             "add symbolic debug info",
             `$(WINDOWS
-                Add CodeView symbolic debug info with
-                $(LINK2 $(ROOT_DIR)spec/abi.html#codeview, D extensions)
```
Comment (CyberShadow) [inline src/dmd/cli.d]: Sorry, why remove the link to https://dlang.org/spec/abi.html#codeview ?

Comment (rainers) [reply src/dmd/cli.d]: I removed the "extension" part as that only applies to OMF and no debugger really supports them (mago can interpret them, but you normally run the exe through cv2pdb that translates them back to something a C++ debugger can deal with), so it's not really relevant anymore.

**Review by CyberShadow — APPROVED** [review]

**Comment by CyberShadow** [comment]
Force-merging to override transient (hopefully) CI failure on master.

**Comment by rainers** [comment]
> Force-merging to override transient (hopefully) CI failure on master.

Thanks. I hope I haven't screwed up when just force-pushing.

**Comment by MoonlightSentinel** [comment]
> ```
> ERROR: dmd/frontend.d(147): Deprecation: function `std.algorithm.iteration.each!(addGlobalIdent).each!(const(string[])).each` is deprecated
> ```
> 
> I cannot reproduce this locally and don't see any related recent changes. Anyone know what's going on?

This error message was caused by one iteration of my PR #10677. Makes me wonder if that build was used somehow ...

**Pull request merged**